### ISSUE TITLE
CyanogenMod 12 ( 5.0.2 )

### ISSUE THREAD

**Issue by MisieQ** [erickok/transdroid#190]
When upload torrent with transdroid for rtorrent apk gone fc

**Comment by erickok** [comment]
Could you supply the crash log (using the Error Report feature in the Help menu of Transdroid)?

**Comment by MisieQ** [comment]
Please describe your problem:

Transdroid 2.3.0 (217)
rTorrent settings:
http://misieq@misieqhba.pl:80/rutorrent/plugins/httprpc/action.php

Connection and error log:
236 -- Thu Jan 29 16:51:20 CET 2015 -- 3 -- Boot signal received, starting
server and rss checker background services
237 -- Thu Jan 29 16:51:20 CET 2015 -- 3 -- Calling d.multicall with params
[ main d.get_hash= d.get_name= d.get_state= d.get_down_rate= d.get_up_rate=
d.get_peers_connected= d.... ]
238 -- Thu Jan 29 16:51:20 CET 2015 -- 3 -- Boot signal received, starting
app update checker service
239 -- Thu Jan 29 16:51:30 CET 2015 -- 3 -- Calling d.multicall with params
[ main d.get_hash= d.get_name= d.get_state= d.get_down_rate= d.get_up_rate=
d.get_peers_connected= d.... ]
240 -- Thu Jan 29 16:51:30 CET 2015 -- 3 -- Skip the update service, as we
already checked the last 24 hours (or to be exact at Thu Jan 29 11:46:19
CET 2015).
241 -- Thu Jan 29 16:51:34 CET 2015 -- 3 -- SeedBox: Retrieved torrent
listing
242 -- Thu Jan 29 16:51:34 CET 2015 -- 3 -- SeedBox: 1 new torrents, 0
newly finished torrents.
243 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
java.lang.IllegalArgumentException: Expected file scheme in URI:
content://media/external/video/media/6802
244 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
java.io.File.checkURI(File.java:225)
245 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
java.io.File.<init>(File.java:177)
246 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
org.transdroid.daemon.Rtorrent.RtorrentAdapter.executeTask(RtorrentAdapter.java:159)
247 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
org.transdroid.daemon.task.DaemonTask.execute(DaemonTask.java:79)
248 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
org.transdroid.core.gui.TorrentsActivity.addTorrentByFile(TorrentsActivity.java:984)
249 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
org.transdroid.core.gui.TorrentsActivity_.access$801(TorrentsActivity_.java:42)
250 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
org.transdroid.core.gui.TorrentsActivity_$9.execute(TorrentsActivity_.java:366)
251 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
org.androidannotations.api.BackgroundExecutor$Task.run(BackgroundExecutor.java:393)
252 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:422)
253 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
java.util.concurrent.FutureTask.run(FutureTask.java:237)
254 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152)
255 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265)
256 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
257 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
258 -- Thu Jan 29 16:52:20 CET 2015 -- 6 -- E:
java.lang.Thread.run(Thread.java:818)
259 -- Thu Jan 29 16:52:26 CET 2015 -- 3 -- Boot signal received, starting
server and rss checker background services
260 -- Thu Jan 29 16:52:26 CET 2015 -- 3 -- Calling d.multicall with params
[ main d.get_hash= d.get_name= d.get_state= d.get_down_rate= d.get_up_rate=
d.get_peers_connected= d.... ]
261 -- Thu Jan 29 16:52:26 CET 2015 -- 3 -- Boot signal received, starting
app update checker service
262 -- Thu Jan 29 16:52:30 CET 2015 -- 3 -- Calling d.multicall with params
[ main d.get_hash= d.get_name= d.get_state= d.get_down_rate= d.get_up_rate=
d.get_peers_connected= d.... ]
263 -- Thu Jan 29 16:52:30 CET 2015 -- 3 -- Skip the update service, as we
already checked the last 24 hours (or to be exact at Thu Jan 29 11:46:19
CET 2015).
264 -- Thu Jan 29 16:52:31 CET 2015 -- 3 -- SeedBox: Retrieved torrent
listing
265 -- Thu Jan 29 16:52:31 CET 2015 -- 3 -- SeedBox: 0 new torrents, 0
newly finished torrents.
266 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
java.lang.IllegalArgumentException: Illegal character in path at index 98:
content://com.ghisler.android.TotalCommander.files/storage/emulated/0/Download/PublicSexAdventures
- Alie - Fucking A Sporty Girl Outdoors-[rarbg.com].torrent
267 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
java.net.URI.create(URI.java:730)
268 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
org.transdroid.daemon.Rtorrent.RtorrentAdapter.executeTask(RtorrentAdapter.java:159)
269 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
org.transdroid.daemon.task.DaemonTask.execute(DaemonTask.java:79)
270 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
org.transdroid.core.gui.TorrentsActivity.addTorrentByFile(TorrentsActivity.java:984)
271 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
org.transdroid.core.gui.TorrentsActivity_.access$801(TorrentsActivity_.java:42)
272 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
org.transdroid.core.gui.TorrentsActivity_$9.execute(TorrentsActivity_.java:366)
273 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
org.androidannotations.api.BackgroundExecutor$Task.run(BackgroundExecutor.java:393)
274 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:422)
275 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
java.util.concurrent.FutureTask.run(FutureTask.java:237)
276 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152)
277 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265)
278 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
279 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
280 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
java.lang.Thread.run(Thread.java:818)
281 -- Thu Jan 29 16:53:18 CET 2015 -- 3 -- Boot signal received, starting
server and rss checker background services
282 -- Thu Jan 29 16:53:18 CET 2015 -- 3 -- Calling d.multicall with params
[ main d.get_hash= d.get_name= d.get_state= d.get_down_rate= d.get_up_rate=
d.get_peers_connected= d.... ]
283 -- Thu Jan 29 16:53:18 CET 2015 -- 3 -- Boot signal received, starting
app update checker service
284 -- Thu Jan 29 16:53:20 CET 2015 -- 3 -- Calling d.multicall with params
[ main d.get_hash= d.get_name= d.get_state= d.get_down_rate= d.get_up_rate=
d.get_peers_connected= d.... ]
285 -- Thu Jan 29 16:53:20 CET 2015 -- 3 -- Skip the update service, as we
already checked the last 24 hours (or to be exact at Thu Jan 29 11:46:19
CET 2015).
286 -- Thu Jan 29 16:53:21 CET 2015 -- 3 -- SeedBox: Retrieved torrent
listing
287 -- Thu Jan 29 16:53:21 CET 2015 -- 3 -- SeedBox: 0 new torrents, 0
newly finished torrents.
29 sty 2015 12:57 "Eric Kok" <notifications@github.com> napisał(a):

**Comment by MisieQ** [comment]
Its error log :) im not use cm file menager because totcmd its better :)
29 sty 2015 16:54 "Michał Duda" <misieqhba@gmail.com> napisał(a):

**Comment by kecksk0enig** [comment]
266 -- Thu Jan 29 16:53:12 CET 2015 -- 6 -- E:
java.lang.IllegalArgumentException: Illegal character in path at index 98:
content://com.ghisler.android.TotalCommander.files/storage/emulated/0/Download/PublicSexAdventures
- Alie - Fucking A Sporty Girl Outdoors-[rarbg.com].torrent

IDK but index 98 is right in front of  '-Alie '
may be a linebrake (or did the bugtracker do some fancy linebreaking?)

**Comment by MisieQ** [comment]
Test witch other torrent, name no have special chars but still fc ;(
Log:http://pastebin.com/sdSi3jw1
29 sty 2015 19:50 "kecksk0enig" <notifications@github.com> napisał(a):

**Comment by MisieQ** [comment]
If you need i can create account on seedbox and you can test
 29 sty 2015 21:22 "Michał Duda" <misieqhba@gmail.com> napisał(a):

**Comment by kecksk0enig** [comment]
its another error than before 
```
302 -- Thu Jan 29 21:13:28 CET 2015 -- 6 -- E: java.lang.IllegalArgumentException: Expected file scheme in URI: content://com.ghisler.android.TotalCommander.files/storage/emulated/0/Download/FakeTaxi.e199.Lucy.torrent
```
which looks like something over here [Stackoverflow](https://stackoverflow.com/questions/559902/android-how-can-i-convert-android-net-uri-object-to-java-net-uri-object)

**Comment by erickok** [comment]
Should be fixed in a3a595d9.
